src/widgets/aboutDialog.ui

```diff
@@ -28,7 +28,8 @@ Eric Zhang
 Georgi Georgiev
 Sabri Ünal
 Mathieu Bousquet
-Yaron Shahrabani</property>
+Yaron Shahrabani
+K.B.Dharun Krishna</property>
```

Comment:
re. commit message, can you change it to `aboutDialog: Add @kbdharun to credits`?